MovieChecker/MovieChecker/MovieList.m

```diff
@@ -0,0 +1,81 @@
+- (void)fetchMovies
+{
+    NSURL *url = [RottenTomatoesHelperMethods boxOfficeURL];
+    NSURLSessionDataTask *dataTask = [self.session
+                                      dataTaskWithURL:url
+                                      completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
+                                          if (error) {
+                                              // TODO: Handle the error!
+                                              return;
+                                          }
+                                          NSHTTPURLResponse *httpResp = (NSHTTPURLResponse *)response;
+                                          if (httpResp.statusCode != 200) {
+                                              // TODO: Handle the bad status code!
+                                              return;
+                                          }
+                                          self.movies = [RottenTomatoesHelperMethods interpretBoxOfficeMoviesFromData:data
+                                                                                                withkey:@"movies"];
+                                          [[NSOperationQueue mainQueue] addOperationWithBlock:^{
+                                              [self.delegate synchMoviesWithArray:self.movies];
+                                          }];
+                                      }];
+    [dataTask resume];
+}
+
+- (void)downloadImageForMovie:(Movie *)movie
+{
+    NSURL *url = [NSURL URLWithString:movie.imagePath];
+    NSURLSessionDataTask *downloadImageTask = [[NSURLSession sharedSession]
+                                               dataTaskWithURL:url
+                                               completionHandler:^(NSData *data,
+                                                                   NSURLResponse *response,
+                                                                   NSError *error) {
+                                                   if (error) {
+                                                       // TODO: Handle the error!
+                                                       return;
+                                                   }
+                                                   NSHTTPURLResponse *httpResp = (NSHTTPURLResponse *)response;
+                                                   if (httpResp.statusCode != 200) {
+                                                       // TODO: Handle the bad status code!
+                                                       return;
+                                                   }
+                                                   movie.image = [[UIImage alloc] initWithData:data];
+                                               }];
+    [downloadImageTask resume];
+}
+
+- (id)initWithDelegate:(id)delegate
+{
+    self = [super init];
+    if (self) {
+        _delegate = delegate;
+        _movies = [NSMutableArray array];
```

Comment:
The `movies` property isn't a mutable array; it also doesn't look like you'd make use of whatever value is saved here anyway.  Is there any need to set it at all?